kubernetes_platform/python/kfp/kubernetes/toleration.py

```diff
@@ -40,7 +40,7 @@ def add_toleration(
             Pipeline task.
         key:
             key is the taint key that the toleration applies to. Empty means
-            match all taint keys. If the key is empty, operator must be Exists;
+            match all taint keys. If the key is empty, operator must Exist;
```

Comment:
I think `Exists` is correct since it's a value. See the function argument `operator`.

Reply:
🤦🏾